src/xrpld/rpc/detail/RPCHelpers.cpp

```diff
@@ -250,7 +250,10 @@ getAccountObjects(
     {
         // it's possible the user had nftoken pages but no
         // directory entries
-        return mlimit < limit;
+        if (mlimit >= limit)
+            jvResult[jss::account_objects] = Json::arrayValue;
+
```

Comment:
Please add comments why `true` is always returned in this case.

Reply:
updated. also added for  `if (mlimit >= limit)`. please check